.github/workflows/test_comprehensive.yml

```
@@ -43,23 +43,23 @@ jobs:
       fail-fast: false
       matrix:
         platform: [ubuntu-latest, windows-latest]
-        python: ["3.8", "3.9", "3.10"]
+        python: ["3.9", "3.10"]
         backend: [pyqt5, pyside2]
         include:
           - python: 3.9
             platform: macos-latest
             backend: pyqt5
           # test with minimum specified requirements
-          - python: 3.8
+          - python: 3.9
             platform: ubuntu-20.04
             backend: pyqt5
             MIN_REQ: 1
           # test with --async_only
-          - python: 3.8
+          - python: 3.9
             platform: ubuntu-20.04
             toxenv: async-py38-linux-pyqt5
```

thw problem is here. You specify python 3.9 abut still call test on 3.8 (because of `async-py38-linux-pyqt5`. But Ipython 8.13.2 does not support python 3.8 
```suggestion
            toxenv: async-py39-linux-pyqt5
```

---

Rha ! Thanks !!!